model/etcd.go

```diff
@@ -29,6 +31,15 @@ type EtcdDisasterRecovery struct {
 	Automated bool `yaml:"automated,omitempty"`
 }
 
+type EtcdOpts struct {
+	QuotaBackendBytes int `yaml:"quotaBackendBytes,omitempty"`
+}
```

Comment:
How about adding another supported arg in order to test that your code is good with mutliple user provided args?